### ISSUE TITLE
Feature request: Make it work in the LaCros browser

### ISSUE THREAD

**Issue by bkennelly** [google/extra-keyboards-for-chrome-os#129]
This extension is indispensable, but does not function in the LaCros browser.

**Comment by colmbuckley** [comment]
Yes; the chrome.input.ime framework is not available in LaCros, this is apparently an intentional product decision. It is being discussed on the chromium bug forum, but I don't have high hopes that this framework will be brought back :-(

**Comment by saghatelian** [comment]
Does this mean that there will be no way to add extra keyboard layouts to Chrome OS after Lacros is the default browser? I've just turned on the lacros-only flag to be able to have parallel browser sessions in 2 Google accounts. That feature works great, but my custom keyboard layout disappeared, which is a deal-breaker for me.

**Comment by colmbuckley** [comment]
Yes; at present Lacros does not include the necessary Javascript API (chrome.input.ime) to allow keyboard layout extensions to work. This was an intentional design choice, mostly for security reasons (IME is pretty terrible from a security perspective). There is still some discussion about allowing a more limited form of the IME API which would allow most of these extensions to be used (with some rewriting and possibly small functionality losses); but as far as I know a final decision has not yet been made on this.

**Comment by jpalmer** [comment]
Lacros has been deprecated so this now obsolete
